scripts/build_libpq.sh

```diff
@@ -6,21 +6,32 @@ set -euo pipefail
 set -x
 
 OPENSSL_VERSION="1.1.1d"
-LDAP_VERSION="2.4.48"
+LDAP_VERSION="2.4.50"
 SASL_VERSION="2.1.27"
 # If you change this, fix WANT_LIBPQ too in .travis.yml
 POSTGRES_VERSION="11.5"
+# Fetch architecture
+UNAME_M=`uname -m`
 
 yum install -y zlib-devel krb5-devel pam-devel
 
 # Need perl 5.10.0 to build/install openssl
 curl -sL https://install.perlbrew.pl | bash
+
+# Required for aarch64 architecture
+if [[ ${UNAME_M} == "aarch64" ]]; then
+	cp /bin/* ~/perl5/perlbrew/bin/
```

Comment:
Could you explain what is this for please? :slightly_smiling_face:

Reply:
Without copying the binaries in `bin` folder, it is unable to run `mkdir` and some more system commands while running bashrc command.

Reply:
that's surprising, especially given the fact you are setting a PATH after perlbrew. Maybe it's the case to run that build in a separate shell? I can't remember actually what perl is used for (I guess to build openssl though).

I think arm behaves in a very different way than x86: maybe that script is just broken with arm and there is a better way of building what we need?

Reply:
For manylinux2014-aarch64, perl installed from yum is 5.16.3. I have removed the above changes and kept perl installation only for x86.